### ISSUE TITLE
<label> should be before input

### ISSUE THREAD

**Issue by alisonjoseph** [IBM/carbon-components-react#1149]
Related to: https://github.com/IBM/carbon-components/issues/1009

Need to update/check following components

- [x] Number Input
- [x]   Search
- [x]   Select https://github.com/IBM/carbon-components-react/pull/1134
- [x]   Text Input
- [x]   Text Area
- [x] Date Picker
- [ ]
